.env

```diff
@@ -30,11 +30,11 @@
 #DB_PORT=5436
 
 #Kintsugi (testnet)
-ARCHIVE_ENDPOINT=https://api-dev-kintsugi.interlay.io/subsquid-gateway/graphql
-CHAIN_ENDPOINT=wss://api-dev-kintsugi.interlay.io/parachain
-SS58_CODEC="kintsugi"
-BITCOIN_NETWORK='testnet'
-DB_PORT=5433
+# ARCHIVE_ENDPOINT=https://api-dev-kintsugi.interlay.io/subsquid-gateway/graphql
```

Comment:
Please restore `.env` to its original state. There are other processes that rely on the values being set.

Reply:
I fixed this.